addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -253,8 +251,8 @@ func _get_history(buffer: Dictionary, tick: int) -> Dictionary:
 	if buffer.is_empty():
 		return {}
 	
-	var earliest: int = buffer.keys().min()
-	var latest: int = buffer.keys().max()
+	var earliest = buffer.keys().min()
+	var latest = buffer.keys().max()
```

Comment:
Static typing in places such as this help greatly in readability. Without the int (hinting to tick), if you land suddenly on this codeblock, you dont know what buffer.keys() can be, could be variant in the reader's head

Reply:
As discussed earlier, convention is to not use static typing inside methods. Renaming variables to be more expressive of what they do is a good idea though. 

The latter can be done either in a separate PR intended for readability, or later in this PR's lifecycle.

Reply:
Did them in seperate PRs: #306 and #305